dmoj/tests/test_checker.py

```diff
@@ -21,3 +21,24 @@ def test_standard(self):
         # Checkers should handle mixed bytes/str
         assert check(b'a', u'a')
         assert not check(b'a', u'b')
+
+    def test_identical(self):
+        from dmoj.checkers.identical import check
+
+        def is_pe(res, feedback='Presentation Error, check your whitespace'):
+            return res is not True and not res.passed and res.feedback == feedback
+
+        assert check(b'a\nb\nc', b'a\nb\nc')
+        assert check(b'a\nb\nc', b'a\nb\nc')
+        assert is_pe(check(b'a \nb\nc', b'a\nb\nc'))
+        assert is_pe(check(b'a\nb\nc', b'a\nb\nc\n'))
+        assert is_pe(check(b'a\nb\nc', b'a\nb\nc\n', pe_allowed=False), feedback=None)
+
+    def test_unordered(self):
+        from dmoj.checkers.unordered import check
```

Comment:
Why are you adding unit tests for a checker that's deprecated... in this very change?